src/App.js

```diff
@@ -1,21 +1,22 @@
-//import Navigation from './components/Navigation.Js';
+// import Navigation from './components/Navigation.Js';
```

Comment:
You did great job! App is working as expected!

- [ ] However, we are not allowed to use commented code in this project. Kindly remove this line!